src/components/LeftSidebar/ConversationsList/Conversation.vue

```diff
@@ -93,13 +93,27 @@
 				{{ t('spreed', 'Delete conversation') }}
 			</NcActionButton>
 		</template>
+		<template v-else-if="item.token" slot="actions">
+			<NcActionButton icon="icon-clippy"
+				@click.stop.prevent="handleCopyLink">
+				<template #icon>
```

Comment:
**suggestion (blocking):** `icon="icon-clippy"` is redundant here, it is overridden by the `#icon` slot.

```suggestion
			<NcActionButton @click.stop.prevent="handleCopyLink">
				<template #icon>
```

Reply:
In case we want to open the chat on join click:

```suggestion
			<NcActionButton close-after-click @click="onClick">
				<template #icon>
```

Reply:
I will pick the `close-after-click `